metadata-ingestion/src/datahub/ingestion/source/mlflow.py

```diff
@@ -277,7 +276,11 @@ def _get_dataset_schema(self, schema: str) -> Optional[List[Tuple[str, str]]]:
         try:
             schema_dict = json.loads(schema)
         except json.JSONDecodeError:
-            print("Failed to parse schema JSON")
+            self.report.warning(
+                title="Failed to parse schema JSON",
+                message=f"Schema: {schema}",
+                context="Adding schema information as a custom property instead",
+            )
```

Comment:
it'd also be nice to include the dataset name in the context field

```suggestion
            self.report.warning(
                title="Failed to load dataset schema",
                message="Schema metadata will be missing due to a JSON parsing error.",
                context=f"Schema: {schema}",
            )
```

Reply:
nice catch, done